src/PKSim.Infrastructure/ProjectConverter/v7_3/Converter721To730.cs

```diff
@@ -44,19 +53,44 @@ public Converter721To730(ICompoundFactory compoundFactory, ICloner cloner, ICrea
 
       public void Visit(Compound compound)
       {
+         resetAllParametersToIsDefaultIn(compound);
          convertCompound(compound);
       }
 
       public void Visit(Simulation simulation)
       {
+         simulation.AllBuildingBlocks<IPKSimBuildingBlock>().Each(resetAllParametersToIsDefaultIn);
          simulation.Compounds.Each(convertCompound);
+         resetAllParametersToIsDefault(simulation.All<IParameter>());
       }
 
       public void Visit(PKSimProject project)
       {
          convertProject(project);
       }
 
+      public void Visit(IPKSimBuildingBlock buildingBlock)
+      {
+         resetAllParametersToIsDefaultIn(buildingBlock);
+      }
+
+      private void resetAllParametersToIsDefaultIn(IContainer container)
+      {
+         resetAllParametersToIsDefault(container.GetAllChildren<IParameter>());
+      }
+
+      private void resetAllParametersToIsDefault(IEnumerable<IParameter> allParameters)
+      {
```

Comment:
would rename the function to smthg like `adjustDefaultStateOfAllParameters`
(because `resetAllParametersToIsDefault `lets me thinks that all parameters get `IsDefault=true`)

Reply:
Done